src/app/infrastructure/computerScore.js

```diff
@@ -0,0 +1,25 @@
+export const computerScore = (level, numberOfQuestions) => {
```

Comment:
Moim zdaniem zapis przedstawiony poniżej (z użyciem mapowania) jest bardziej przejrzysty niż `switch/case`:
```js
const getRandomNumberGeneratorByLevel = {
    1: () => Math.floor(Math.random() * 34), //0-33
    2: () => 34 + Math.floor(Math.random() * 33), //34-66
    3: () => 67 + Math.floor(Math.random() * 34), //67-100
};

export const computerScore = (level, numberOfQuestions) => {
    if (numberOfQuestions <= 0) throw new Error('NumberOfQuestions should to be greater than 0');
    if (!(level in getRandomNumberGeneratorByLevel)) throw new Error('Argument level should to be a number 1,2 or 3');
    const randomNumber = getRandomNumberGeneratorByLevel[level]();
    return Math.floor(numberOfQuestions * (randomNumber / 100)) * 5;
};
```
Po drugie jak chcemy coś zwalidować to lepiej jest to zrobić zaprzeczeniem `if not`:
```js
if(!(numberOfQuestions > 0)) // lub poprostu numberOfQuestions <= 0
     throw new Error('NumberOfQuestions should to be greater than 0');
```
niż `if/else`:
```js
if (numberOfQuestions > 0) {
     ...
 } else {  
     throw new Error('NumberOfQuestions should to be greater than 0');
 }
 ```